player.py

```diff
@@ -399,7 +402,7 @@ def set_ability_costs(self):
         self.rake_cost = 35. / (1 + self.berserk)
         self.mangle_cost = self._mangle_cost / (1 + self.berserk)
         self.bite_cost = 35. / (1 + self.berserk)
-        self.rip_cost = 30. / (1 + self.berserk)
+        self.rip_cost = 30. / (1 + self.berserk) - 10 * self.t10_2p_bonus
```

Comment:
Do we know for sure that the reduced Energy cost is applied _after_ the factor of 2 discount during Berserk, versus before?

Reply:
It should be before